core/controlplane/config/encrypted_assets.go

```diff
@@ -482,12 +482,18 @@ func ReadRawAssets(dirname string, manageCertificates bool, caKeyRequiredOnContr
 		path := filepath.Join(dirname, file.name)
 		data, err := RawCredentialFileFromPath(path, file.defaultValue)
 		if err != nil {
-			return nil, fmt.Errorf("Error reading credential file %s: %v", path, err)
+			return nil, fmt.Errorf("error reading credential file %s: %v", path, err)
```

Comment:
Yep, an error message should start with a lower case character! Thx for starting to conform that.